src/main/java/com/redbrokers/processing/model/Execution.java

```diff
@@ -1,14 +1,26 @@
 package com.redbrokers.processing.model;
 
-import javax.persistence.CascadeType;
-import javax.persistence.FetchType;
-import javax.persistence.JoinColumn;
-import javax.persistence.OneToMany;
+import lombok.Getter;
+import lombok.Setter;
+
+import javax.persistence.*;
 import java.math.BigDecimal;
 import java.time.LocalDateTime;
-
+@Entity
+@Getter
+@Setter
 public class Execution {
+    @Id
+    @Column(name = "id", nullable = false)
+    private Long id;
```

Comment:
I was thinking making all the id's of type UUID would be more uniform?
And also, the db schema's we have has all the id's as UUID's though

Reply:
ok